spec/controllers/groups_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe GroupsController do
+  describe "GET /groups" do
+    context "when invalid params" do
+      it "does not create a new group" do
+        expect do
+          post :create, params: { group: { name: "St. Paul" } }
```

Comment:
I think you need a line between lines 6 and 7

Reply:
Should I?? 🤔 I think all my other specs are like this... thanks! 😃

Reply:
I'm not a hundred percent sure lol, maybe if rubocop doesn't comment then leave it